containers/images/pulp/container-assets/route_paths.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import pkgutil
+import json
+import re
+import sys
+import nginx
+from pathlib import Path
+
+nginx_configs = []
+
+for i in pkgutil.iter_modules():
+    # filter pulp and galaxy packages
+    if re.search(r'pulp_.*|galaxy_ng',i.name):
+        # only add the filename to array if it exists
+        if Path(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf").is_file():
+            nginx_configs.append(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf")
+
+name = sys.argv[1]
+router = []
+
+for plugin_conf in nginx_configs:
+    app = "galaxy" if "galaxy" in plugin_conf else plugin_conf.split("pulp_")[1].split("/")[0]
+    conf = nginx.loadf(plugin_conf)
+    for location in conf.filter("Location"):
+        path = location.value
+        if not path.startswith("/"):
+            path = f"/{path}"
+        rewrite = ""
+        for key in location.keys:
+            target_port = ""
+            svc_name = ""
+            if key.name.strip("'") == "proxy_pass":
```

Comment:
i am not sure you need to strip, do you?
```

In [29]: conf.filter("Location")[0].keys[4].name=='proxy_pass'
Out[29]: True

In [30]: conf.filter("Location")[0].keys[4].name
Out[30]: 'proxy_pass'

```

Reply:
I'm not sure either. Maybe @fao89 found a snippet where `python-nginx` module returned these "surplus" quotes.
@fao89 WDYT?

Reply:
when I tested it was adding the extra quotes, so I put strip to ensure